app/code/Magento/Catalog/Test/Unit/Model/CategoryTest.php

```diff
@@ -424,39 +441,33 @@ public function testReindexFlatDisabled(
 
     public function testGetCustomAttributes()
     {
-        $interfaceAttributeCode = 'name';
-        $customAttributeCode = 'description';
-
-        $this->resource
-            ->method('getCustomAttributesCodes')
-            ->willReturn([$customAttributeCode]);
-        $this->category->setData($interfaceAttributeCode, "sub");
+        $nameAttributeCode = 'name';
+        $descriptionAttributeCode = 'description';
```

Comment:
why did you rename the variables? For the test it's not important that it's name and description, but that one is an interface attribute and the other is custom